bots.adoc

```diff
@@ -59,7 +62,6 @@ This bot is the first GitHub bot used in the Jenkins Infrastructure.
 ex:
 
 * Automatically open pull request
-* jenkinsadmin irc bot
```

Comment:
We replaced the gihtub account [jenkinsadmin](https://github.com/jenkinsadmin) by [jenkins-infra-bot](https://github.com/jenkins-infra-bot)

Reply:
I am not sure that jenkinsadmin irc bot is refering to, the github account? the irc account ? jenkins-admin

Reply:
the irc account is what I assumed it meant

Reply:
IIRC it was a historical doc I just moved a while ago